build.gradle

```diff
@@ -126,6 +128,8 @@ ext {
 
     glide = 'com.github.bumptech.glide:glide:4.8.0'
 
+    truetime = 'com.github.instacart.truetime-android:library:3.4'
```

Comment:
I'd like to remove truetime from the build, so we don't have to deal with jitpack. Is that a problem?

Reply:
Nope, should be fine.

Reply:
Removed.